pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -350,6 +350,36 @@ func (c *Cluster) checkHealth(ctx context.Context) error {
 		}
 	}
 
+	if c.spec.Network.MultiClusterService.Enabled {
+		for _, mon := range c.ClusterInfo.Monitors {
+			var failMon bool
+			monResourceName := resourceName(mon.Name)
+			isAlreadyExported, err := k8sutil.IsServiceExported(c.ClusterInfo.Context, c.context, monResourceName, c.ClusterInfo.Namespace)
+			if err != nil {
+				return errors.Wrapf(err, "failed to check if the service %q is already exported", mon.Name)
+			}
+			if !isAlreadyExported {
+				failMon = true
+			} else {
+				exportedIP, err := k8sutil.GetExportedServiceIP(fmt.Sprintf("%s.%s.svc.clusterset.local", monResourceName, c.ClusterInfo.Namespace))
```

Comment:
As discussed, no need to check for conditions (e.g. changing IP) where the mon failover would anyway go out of quorum and naturally be failed over.

Reply:
removed this condition.